modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+** The number of Argo CD instances in a given state, such as pending, available, among others. 
+** The total time for the reconciliation loops to complete per instance,
+** The amount of resources used such as CPU, memory, and number of goroutines.
++
+To access these metrics, go to the *Observe* tab on the web console, and run queries against the monitoring stack. link:https://issues.redhat.com/browse/GITOPS-2645[GITOPS-2645]
++
+[NOTE]
+====
+You need to install the {gitops-title} Operator in the default `openshift-gitops-operator` namespace and have monitoring enabled, in order to have these metrics automatically available.
+====
+
+* Previously, timestamps were presented in a Unix epoch format. With this update, the timestamps are changed to RFC3339 format, for example: 2023-06-27T07:12:48-04:00, to improve overall readability. link:https://issues.redhat.com/browse/GITOPS-2898[GITOPS-2898]
+
+* With this update, the default Argo CD instance in the `openshift-gitops` namespace has restricted permissions for non-admin users by default. This improves security because non-admin users no longer have access to sensitive information. However, as an administrator, you can set permissions and grant non-admin users access to the resources managed by the default `openshift-gitops` Argo CD instance by configuring your Argo CD RBAC. This change only applies to the default {gitops-title} instance. link:https://issues.redhat.com/browse/GITOPS-3032[GITOPS-3032]
+
+* With this update, the default installation namespace for {gitops-title} Operator has been changed to its own namespace called `openshift-gitops-operator`. You can still choose the old default installation namespace, `openshift-operators`, through a drop-down menu available in the OperatorHub UI at installation time. You can also enable cluster monitoring on the new namespace by selecting the check box, which makes the Operator's performance metrics accessible within the {product-title} web console. link:https://issues.redhat.com/browse/GITOPS-3073[GITOPS-3073]
++
+[NOTE]
+====
+The {gitops-title} Operator's metrics are only available when the Operator is installed in the default namespace, `openshift-gitops-operator`.
+====
+
+* Before this update, there was no option for scaling the application controller replicas dynamically. Now, you can dynamically scale the number of application controllers based on the number of clusters managed by each application controller. link:https://issues.redhat.com/browse/GITOPS-3287[GITOPS-3287]
+
+* Before this update, there was no option for choosing an algorithm for distributing the destination clusters equally across the different application controller shards. Now, you can set the sharding algorithm to `round-robin` parameter, which distributes clusters equally across the different application controller shards so that the synchronization load is spread equally among the shards. link:https://issues.redhat.com/browse/GITOPS-3288[GITOPS-3288]
```

Comment:
```suggestion
* Before this update, there was no option for choosing an algorithm for distributing the destination clusters equally across the different application controller shards. Now, you can set the sharding algorithm to the `round-robin` parameter, which distributes clusters equally across the different application controller shards so that the synchronization load is spread equally among the shards. link:https://issues.redhat.com/browse/GITOPS-3288[GITOPS-3288]
```

Reply:
Done!